scripts/docuCheck.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e # exit on errors
+
+BASE_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+echoe() {
```

Comment:
Maybe is a good idea to have this function in a separate file to avoid duplication, as we are using it already in a couple of scripts and we will probably use it more in the future.

Reply:
done